docs/design/autoware-concepts/difference-from-ai-and-auto.md

```diff
@@ -0,0 +1,37 @@
+# How is Autoware Core/Universe different from Autoware.AI and Autoware.Auto?
+
+Autoware is the world's first "all-in-one" open-source software for self-driving vehicles.
+Since it was first released in 2015, there have been multiple releases made with differing underlying concepts, each one aimed at improving the software.
+
+## Autoware.AI
+
+[Autoware.AI](https://github.com/Autoware-AI/autoware.ai) is the first distribution of Autoware that was released based on ROS 1. The repository contains different packages to cover different aspects of autonomous driving technologies, including sensing, actuation, localization, mapping, perception, planning, and so on.
+
+While it was successful in attracting many developers and contributions, there were no particular use-cases nor an ODD defined for the software. It also lacked design documents to specify architecture, and each package had different coding standards with very low test coverage.
```

Comment:
I guess the part `there were no particular use-cases nor an ODD defined for the software` is a bit difficult to understand, but I have no alternative idea yet. :thinking:

Reply:
We could explain what those terms mean but then this page will get very long! We could add a glossary of terms like `use-case` and `ODD` to the documentation or one of the wikis and then link to that?

Reply:
I think we don't have the pages yet. :cry: 
Instead, I think it's good to add a supplemental explanation of what happened by "there were no particular use-cases nor an ODD defined for the software".
For example, the development efficiency was bad, etc.

Reply:
How about this? It includes the reasoning for the Autoware architectural proposal from TIER IV, so should be good!

```suggestion
While it was successful in attracting many developers and contributions, it was difficult to improve Autoware.AI's capabilities for a number of reasons:

- A lack of concrete architecture design leading to a lot of built-up technical debt, such as tight coupling between modules and unclear module responsibility.
- Differing coding standards for each package, with very low test coverage.

Furthermore, there was no clear definition of the conditions under which an Autoware-enabled autonomous vehicle could operate, nor of the use cases or situations supported (eg: the ability to overtake a stationary vehicle). 
```

Reply:
So nice. :smile: Thank you!